client/go/internal/cli/cmd/status_test.go

```diff
@@ -141,6 +141,23 @@ func TestStatusCloudDeployment(t *testing.T) {
 	stdout.Reset()
 	client := &mock.HTTPClient{}
 	cli.httpClient = client
+	// Deployment in progress, with implicit fast wait
+	client.NextResponse(mock.HTTPResponse{
+		URI:    "/application/v4/tenant/t1/application/a1/instance/i1/job/dev-us-north-1?limit=1",
+		Status: 200,
+		Body:   []byte(`{"runs": [{"id": 1337}]}`),
+	})
+	client.NextResponse(mock.HTTPResponse{
+		URI:    "/application/v4/tenant/t1/application/a1/instance/i1/job/dev-us-north-1/run/1337?after=-1",
+		Status: 200,
+		Body:   []byte(`{"active": true, "status": "running"}`),
+	})
+	assert.NotNil(t, cli.Run("status", "deployment"))
+	assert.Equal(t, `Timed out waiting for deployment to converge. See https://console.vespa-cloud.com/tenant/t1/application/a1/dev/instance/i1/job/dev-us-north-1/run/1337 for more details
```

Comment:
Couldn't we return a message more like "deployment started" or "deployment running", and then the "use wait" hint? Obviously only on the invocation where we actually start a new job.

Reply:
... or is this just a different API? I guess maybe it is. But the messages are the same?

Reply:
Changed the message.